avocado/core/parser.py

```diff
@@ -46,6 +51,32 @@ def _get_option_tuples(self, option_string):
         return []
 
 
+class StdoutAlreadyClaimedError(Exception):
```

Comment:
Oops, I introduced this exception, but decided to go with `exceptions.OptionValidationError`.  Will remove it in v2.